### ISSUE TITLE
Labeling adjacent squares in surveyor sheets*

### ISSUE THREAD

**Issue by dbl3raf** [bird-team/brisbane-bird-atlas#111]
the latest batch of surveyor sheet maps uses hybrid maps - my reasoning was that it can be tricky to identify where many grid squares are, and that the labelling on the hybrid maps would help readers orientate, especially to some of the place names mentioned in the descriptions. But the extra labelling makes some of the maps a bit cluttered. Any thoughts @Louis-Backstrom ?

**Comment by jeffreyhanson** [comment]
Hmm, we could add an extra column into the `grid-metadata.xlsx` file that indicates which type of map (e.g. satellite, hybrid, etc) each surveyor sheet should contain?

**Comment by Louis-Backstrom** [comment]
I agree that some labelling to help the reader orientate would be good, but I think it is too cluttered at the moment. We've already got cluttering problems with some maps and I think this would make it worse. The only other thought I have with maps to help the reader orientate would be to have the adjoining squares labelled on the map so they know the region - something like this?

![image](https://user-images.githubusercontent.com/44354406/49693968-a54d3200-fbcc-11e8-9989-8b87c8cfdea9.png)

Excuse the horrendous mouse on paint handwriting...

**Comment by dbl3raf** [comment]
Labels for adjacent grid squares would be great if this is easy to program by @jeffreyhanson - and I love the suggestion of having a column where we could specify which map type each surveyor sheet should have. I think that squares with less survey effort really benefit from having hybrid maps, e.g. http://github.com/bird-team/brisbane-bird-atlas/releases/download/v.0.0.2/grid-10050-albany-creek.pdf

**Comment by jeffreyhanson** [comment]
I'll have a go this weekend and see if I can add the column to specify the map type for each surveyor sheet. But adding labels to adjacent grid squares will be a lot trickier.

**Comment by jeffreyhanson** [comment]
I have updated the code to use the `map_type` column in the grid metadata spreadsheet for the map types. However, it does not currently work because someone has enforced data types on the spreadsheet, and for what ever reason, Libre Office is treating the new column as a boolean or number - and this means that R is reading in the "hybrid" labels as 1s or missing values. Can someone please set the cell types to text or remove cell types from the spreadsheet?

**Comment by jeffreyhanson** [comment]
I have also added a section to the wiki on changing the map types.

**Comment by Louis-Backstrom** [comment]
I've just tried to change all the cell types to text in the grid metadata spreadsheet - should this fix the build errors?

**Comment by jeffreyhanson** [comment]
Thanks for trying - but that didn't fix it. I'll just export it as a CSV then re-export as a XLSX.

**Comment by dbl3raf** [comment]
awesome, thanks @jeffreyhanson  - I'll have a play when the build is working again

**Comment by Louis-Backstrom** [comment]
Is this issue resolved for now? Can we hibernate it until @jeffreyhanson has time to play around with adding labels to adjacent grid squares?

**Comment by jeffreyhanson** [comment]
Hmm, we have the functionality for hybrid maps (as per the original motivation in this thread), but we don't have the labels for adjacent grid cells that were suggested later in the issue - I suggest we hibernate it for now?

**Comment by Louis-Backstrom** [comment]
Agreed. Done
